src/App.js

```diff
@@ -1,16 +1,46 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import './components/ChatLog.css';
 
 const App = () => {
+  const [messages, setMessages] = useState(chatMessages);
+
+  const onClickLikeButton = (updatedMessage) => {
+    const newMessages = messages.map((message) => {
+      if (message.id === updatedMessage.id) {
+        return updatedMessage;
+      } else {
+        return message;
+      }
+    });
+    setMessages(newMessages);
```

Comment:
I might consider refactoring this, so that it doesn't update ALL of the message data, just the `liked` property